config/crd/bases/appstudio.redhat.com_enterprisecontractpolicies.yaml

```diff
@@ -86,27 +86,9 @@ spec:
                   integration
                 type: string
               sources:
-                description: Sources of the policy
+                description: One or more source urls for the policy rules
```

Comment:
Do we need to expand on the docs at all saying all supported go-getter style urls or something similar?